airbyte-integrations/connectors/destination-snowflake/gradle.properties

```diff
@@ -1 +1 @@
-JunitMethodExecutionTimeout=30 m
+JunitMethodExecutionTimeout=5 m
```

Comment:
the sum of all the tests runs in about 5 minutes. No single test should take more than 5 minutes, The reason some were timing out at 30minutes was because of all the extra table in the destination database. Now, we're cleaning those as we run tests, guaranteeing that overall the tests will be fast